base/gfspvgmgr/virtual_group_manager.go

```diff
@@ -349,8 +351,6 @@ func (vgm *virtualGroupManager) refreshMetaByChain() {
 // PickVirtualGroupFamily pick a virtual group family(If failed to pick,
 // new VGF will be automatically created on the chain) in get create bucket approval workflow.
 func (vgm *virtualGroupManager) PickVirtualGroupFamily() (*vgmgr.VirtualGroupFamilyMeta, error) {
-	vgm.mutex.RLock()
-	defer vgm.mutex.RUnlock()
```

Comment:
Why remove this lock guard.  vgfIDToVgf may be update/read concurrently in pickVirtualGroupFamily.

Reply:
put it inside the `genVgfFilter`, because call to chain might take too long and it would lock the resource.